src/PlusDataCollection/USDigitalEncodersTracking/vtkPlusUSDigitalEncodersTracker.h

```diff
@@ -146,11 +146,11 @@ class vtkPlusDataCollectionExport vtkPlusUSDigitalEncodersTracker : public vtkPl
 
 
 private:  // Variables.
-    bool coreXY;
+    bool coreXY = false;
     bool configurationViaSerialNumbers = false;
-    long NumberOfEncoders; // Number of connected A2 Encoders
-    long COMPort;
-    long BaudRate;
+    long NumberOfEncoders = 0; // Number of connected A2 Encoders
+    long COMPort = 0;
+    long BaudRate = 9600;

```

Comment:
The most similar other device is PlusCivcoBrachyStepper, as they both use USDigital's DEIDrv32. But merging common parts into a common ancestor class would be some work, with a question: would that make sense?